Jenkinsfile-Internal-Release

```diff
@@ -134,11 +134,12 @@ def updateRepositoryManagerVersion(dockerFileLocation) {
   }
   else {
     // default URL
-    def defaultUrl = /https:\/\/download-staging.sonatype.com\/nexus\/3\/nexus-\$\{NEXUS_VERSION\}-unix\.tar\.gz/
+    def defaultUrl = /https:\/\/download-staging.sonatype.com\/nexus\/3\/nexus-\$\{NEXUS_VERSION\}-\$\{JAVA_VERSION\}-unix\.tar\.gz/
```

Comment:
Why does the unix tarball which doesn't include a java version have a qualifier?

Reply:
This is likely to break any automation that folks have for downloading the tarball